content/docs/2.0/concepts/scaling-deployments.md

```diff
@@ -163,7 +163,7 @@ advanced:
 
 **`horizontalPodAutoscalerConfig:`**
 
- This section contains configuration that is the same as some parts of the standard HorizontalPodAutoscaler configuration. KEDA would feed properties from this section into a apropriate places into the HPA configuration. This way one can modify the HPA that is being created and managed by KEDA.
+ This section contains configuration that is the same as some parts of the standard horizontal pod autoscaler (HPA) configuration. KEDA would feed properties from this section into a appropriate places into the HPA configuration. This way one can modify the HPA that is being created and managed by KEDA.
```

Comment:
the kubernetes object is called `HorizontalPodAutoscaler` it is not just some horizontal pod autoscaler. If you want to get rid of the long word, it should be `Horizontal Pod Autoscaler` :)